igvc_ws/src/sensors/src/sensors_webhook.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import socket
+import fcntl
+import struct
+import os
+import json as j
```

Comment:
I wouldn't import json as j. json is short enough and it generally a bad idea to import it to something so generic and commonly used in loops.